src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAO.scala

```diff
@@ -154,7 +154,67 @@ class PostgresDirectoryDAO(protected val dbRef: DbReference,
     */
   override def removeGroupMember(groupId: WorkbenchGroupIdentity, removeMember: WorkbenchSubject): IO[Boolean] = ???
 
-  override def isGroupMember(groupId: WorkbenchGroupIdentity, member: WorkbenchSubject): IO[Boolean] = ???
+  override def isGroupMember(groupId: WorkbenchGroupIdentity, member: WorkbenchSubject): IO[Boolean] = {
+    val gm = GroupMemberTable.syntax("gm")
+
+    val memberClause: SQLSyntax = member match {
```

Comment:
I think you can refactor this to use sub-query helper method we just merged:  

https://github.com/broadinstitute/sam/blob/4e05af7571fd061b9d93dcb2f90131ecb56f5fbf/src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAO.scala#L240

Or I made comments to replace the individual case statement blocks with calls to the specific helper methods, but probably works better just to reuse `workbenchGroupIdentityToGroupPK`